views/settings.php

```diff
@@ -1,100 +1,107 @@
+
+                        <p class="description">
+                            You can find your API Key on your <a href="https://app.bugsnag.com">Bugsnag Dashboard</a>.
+                        </p>
+                    </td>
+                </tr>
+
+                <tr valign="top">
+                    <th>
+                      <label for="bugsnag_notify_severities">Notify Bugsnag About</label>
+                    </th>
+                    <td>
+                        <select name="bugsnag_notify_severities" id="bugsnag_notify_severities">
+                            <?php $this->renderOption("Crashes &amp; errors", "fatal,error", $this->notifySeverities); ?>
+                            <?php $this->renderOption("Crashes, errors &amp; warnings", "fatal,error,warning", $this->notifySeverities); ?>
+                            <?php $this->renderOption("Crashes, errors, warnings &amp; info messages", "fatal,error,warning,info", $this->notifySeverities); ?>
+                        </select>
+                    </td>
+                </tr>
+
+                <!-- Filter Fields -->
+                <tr valign="top">
+                    <th>
+                      <label for="bugsnag_filterfields">Bugsnag Field Filter</label>
+                    </th>
+                    <td>
+                        <textarea id="bugsnag_filterfields" name="bugsnag_filterfields" class="regular-text filterfields"  style="height: 150px;"><?php echo $this->filterFields; ?></textarea>
+                        <p class="description">
+                            The information to remove from Bugsnag reports, one per line.
+                            Use this if you want to ensure you don't send sensitive data such as passwords, and credit card numbers to our servers.
+                        </p>
+                    </td>
+                </tr>
+            </table>
+
+            <div class="submit">
+                <input type="submit" class="button-primary" value="<?php _e('Save Changes') ?>" />
+            </div>
+        <?php endif ?>
+
+        <!-- Common form stuff -->
+        <?php wp_nonce_field('update-options'); ?>
+        <input type="hidden" name="action" value="update" />
+        <input type="hidden" name="page_options" value="bugsnag_api_key,bugsnag_notify_severities,bugsnag_filterfields" />
+    </form>
 </div>
 
+<?php if (!empty($this->apiKey)): ?>
+    <div>
+        <h2>Test your connection to Bugsnag</h2>
+
+        <p>Use this button to test if your API key is able to communicate with Bugsnag.</p>
+
+        <p>This will send a message that can be viewed in <a href="https://app.bugsnag.com">your Bugsnag Dashboard</a>.</p>
+
+        <p>This specifically tests your API key and field filters. Any other settings you have customised will not be used in order to ensure the settings on this page are correct.</p>
```

Comment:
I think we should make this a little simpler:

```suggestion
        <p>Use this button to send a test event that can be viewed in <a href="https://app.bugsnag.com">your Bugsnag Dashboard</a>.</p>

        <p>Note - any <a href="https://docs.bugsnag.com/platforms/php/wordpress/configuration-options/">configuration options</a> applied in code will not be applied to this test event.</p>
```

Reply:
Yeah this is way better